multimap/src/test/java/org/infinispan/multimap/impl/EmbeddedMultimapListCacheTest.java

```diff
@@ -0,0 +1,77 @@
+package org.infinispan.multimap.impl;
+
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.test.SingleCacheManagerTest;
+import org.infinispan.test.data.Person;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.infinispan.functional.FunctionalTestUtils.await;
+import static org.infinispan.multimap.impl.MultimapTestUtils.ELAIA;
+import static org.infinispan.multimap.impl.MultimapTestUtils.JULIEN;
+import static org.infinispan.multimap.impl.MultimapTestUtils.NAMES_KEY;
+import static org.infinispan.multimap.impl.MultimapTestUtils.OIHANA;
+
+/**
+ * Single Multimap Cache Test with Linked List
+ *
+ * @since 15.0
+ */
+@Test(groups = "functional", testName = "multimap.EmbeddedMultimapListCacheTest")
+public class EmbeddedMultimapListCacheTest extends SingleCacheManagerTest {
+
+   EmbeddedMultimapListCache<String, Person> listCache;
+
+   @Override
+   protected EmbeddedCacheManager createCacheManager() throws Exception {
+      // start a single cache instance
+      EmbeddedCacheManager cm = TestCacheManagerFactory.createCacheManager(MultimapSCI.INSTANCE);
+      ConfigurationBuilder builder = new ConfigurationBuilder();
+      cm.createCache("test", builder.build());
+      cm.getClassAllowList().addClasses(Person.class);
+      listCache = new EmbeddedMultimapListCache<>(cm.getCache("test"));
+      return cm;
+   }
+
+   public void testOfferLast() {
```

Comment:
We need some duplicate value tests as well.

Reply:
we can add more coverage, I plan this in future PR (with the other commands implemented as well)
might we improve these coverage in the further pr-s?